dali/customers/test/products/demo.json

```diff
@@ -146,85 +156,106 @@
       },
       "layers": [
+        "ref:refs.Europe_boundaries",
         {
-          "qid": "Temperature-3",
-          "layer_type": "isoband",
-          "parameter": "Temperature",
-          "isobands": "json:isobands/Temperature.json",
-          "css": "isobands/Temperature.css"
-        },
-        {
-          "qid": "Temperature-number-3",
-          "layer_type": "number",
-          "css": "numbers/numbers.css",
-          "parameter": "Temperature",
-          "label": {
-            "dx": 0,
-            "dy": 3,
-            "precision": 0,
-            "missing": "-"
-          },
-          "positions": {
-            "x": 20,
-            "y": 18,
-            "dx": 30,
-            "dy": 30,
-            "ddx": 15
-          }
-        },
-        {
+          "qid": "weathersymbol-1",
+          "producer": "gfs_world",
           "layer_type": "symbol",
+          "css": "symbols/smartweather.css",
+          "symbols": "json:symbols/smartweather.json",
+          "parameter": "smartsymbol",
+          "scale": 1.4,
           "positions": {
             "layout": "latlon",
             "locations": [
+              // Here's an example query to check by coordinates if the producer has the needed data (in this example, Helsinki):
+              // http://smartmet.fmi.fi/timeseries?lonlat=24.941325%2C60.173437&producer=gfs_world&param=utctime,smartsymbol
+              // if the data is missing the smartsymbol won't be drawn.
               {
-                "longitude": 24.93417,
-                "latitude": 60.17556
+                // Helsinki
+                "longitude": 24.941325,
+                "latitude": 60.173437
+              },
+              {
+                // Turku
+                "longitude": 22.264824,
+                "latitude": 60.45451
+              },
+              {
+                // Jyväskylä
+                "longitude": 25.747257,
+                "latitude": 62.242603
+              },
+              {
+                // Rovaniemi
+                "longitude": 25.726967,
+                "latitude": 66.503059
+              },
+              {
+                // Oulu
+                "longitude": 25.469885,
+                "latitude":  65.021545
+              },
+              {
+                // Joensuu
+                "longitude": 29.736916,
+                "latitude":  62.607662
+              },
+              {
+                // Saariselkä
+                "longitude": 27.414838,
+                "latitude":  68.419817
               }
             ],
             "dx": 0,
             "dy": 0
           },
-          "symbol": "dot",
-          "scale": 7
-        },
-        "ref:refs.Europe_boundaries"
+          "attributes": {
+            "id": "weather_1",
+            "class": "Weather"
+          }
+        }
       ]
     },
     {
-      "qid": "v4-",
-      "producer": "ecmwf_maailma_pinta",
+      // 4. same map as map_1 with with time offset.
+      
```

Comment:
```suggestion
```

Remove unnecessary empty lines (not sure if this is the most important thing in the world, but my linters do not like these. Same applies to the comments aswell, but in this case those might be useful.)